zrml/prediction-markets/src/migrations.rs

```diff
@@ -488,7 +609,7 @@ mod tests {
             let market_ids = BoundedVec::<MarketIdOf<Runtime>, CacheSize>::try_from(vec![0, 1])
                 .expect("BoundedVec creation failed");
             let dispute_block = System::current_block_number().saturating_sub(1_u32.into());
-            let dispute_period = <Runtime as crate::Config>::DisputePeriod::get();
+            let dispute_period : <Runtime as frame_system::Config>::BlockNumber = (BLOCKS_PER_DAY as u32).into();
```

Comment:
Don't know why rustfmt doesn't fix this:

```suggestion
            let dispute_period: <Runtime as frame_system::Config>::BlockNumber = (BLOCKS_PER_DAY as u32).into();
```